src/DNS_Mgr.cc

```diff
@@ -442,6 +442,8 @@ void DNS_Mgr::Init()
 
 	if ( ! nb_dns )
 		reporter->Warning("problem initializing NB-DNS: %s", err);
+	else
```

Comment:
Stylistic nitpick: switch if/else branches to have regular case first, and error case second.